azure-kusto-data/src/models.rs

```diff
@@ -1,11 +1,23 @@
+use crate::request_options::RequestOptions;
 use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
 
 #[derive(Debug, Serialize, Deserialize)]
+#[allow(non_snake_case)]
 pub(crate) struct QueryBody {
     /// Name of the database in scope that is the target of the query or control command
     pub db: String,
     /// Text of the query or control command to execute
     pub csl: String,
+    /// Additional parameters and options for fine-grained control of the request behavior
+    pub Properties: Option<RequestProperties>,
```

Comment:
I think here it's better to use a rename directive in serde rather than keep it in a non-standard name

Reply:
yes, done.